src/it/scala/temple/containers/DockerPostgresService.scala

```diff
@@ -0,0 +1,56 @@
+package temple.containers
+
+import java.sql.DriverManager
+
+import com.whisk.docker.{DockerCommandExecutor, DockerContainer, DockerContainerState, DockerKit, DockerReadyChecker}
+import temple.containers.DockerPostgresService.PostgresReadyChecker
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.concurrent.duration._
+
+/** DockerPostgresService encapsulates all configuration for running a postgres docker container */
+object DockerPostgresService {
+  val image            = "postgres:9.6"
+  val databaseUsername = "postgres"
+  val databasePassword = ""
+  val databaseName     = "temple_test"
+  val internalPort     = 5432
+  val externalPort     = 44444
+  val externalUrl      = s"jdbc:postgresql://localhost:$externalPort/$databaseName"
+
+  class PostgresReadyChecker extends DockerReadyChecker {
+
+    // Called when container has started
```

Comment:
This comment is wrong. Should be “called every time, but only succeeds when started”